deploy/kubernetes/crds.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: monitoringdashboards.monitoring.kiali.io
+  annotations:
+    "helm.sh/hook": crd-install
+  labels:
+    app: kiali
```

Comment:
missing version label. We should be consistent across all our objects - use app and version labels